README.md

```diff
@@ -1,65 +1,30 @@
-# Jazz Installer (Developer Preview)
-## Overview
+![Jazz Logo](logo.png)

```

Comment:
We don't have /logo.png in this repository. You may need to commit it to the repository.

Reply:
good catch, fixed it